### PR TITLE
EES-2094 update govuk-frontend

### PR DESCRIPTION
**Pull request by amyb-hiveit**
Updates govuk-frontend from 3.11.0 to 4.7.0.

### Accordion  
Updated to new style component or removed in all places.

release-accordions

Public front end:
-  Release page - updated for additional supporting files, related dashboard & content blocks, removed on help and support section
- Find Statistics page -  release type filter
- Table Tool final step - removed on related information and contact us
- Glossary page - glossary list
- Methodologies page - methodologies list
- Methodology page - content blocks, annexes

Admin:
- Edit release - data and files tab - data uploads (reorderable), data guidance and ancillary file uploads
- Edit release - content tab - updated for additional supporting files, related dashboards and content blocks (reorderable), removed on help and support section
- Edit release - content tab - preview content -  additional supporting files,  related dashboards and content blocks, removed on help and support section
- Edit release - preview release Table Tool - final step - removed on related information and contact us
- Pre-release Table Tool - final step - removed on related information and contact us
- Edit methodology - content tab - content blocks (reorderable) and annexes (reorderable)
- Edit methodology - content tab - preview content   - content blocks  (with copy button) and annexes (with copy button)
- Manage themes and topics 

Some changes have been made to `EditableAccordion` and `ReorderableAccordion` to fix the reordering styles with the new accordions.

### Link styles

Updated link style (currently enabled with a flag, will be enabled by default in next version)

Default:
link1
Hover:
link2

## Other changes

- Updated header HTML - mobile navigation changes and changed logo to SVG (also fixes inconsistent focus style)
- Removed our version of the pagination styles as it's now part of the design system.
- Updated error summary HTML to improve screen reader compatibility 
- Add `data-nosnippet` attribute to cookie banner
- Replaced our text-align classes with govuk versions
- Replaced our display classes with govuk versions
- Update the HTML for hints - change span to div as no longer has display block
- Changed banner links to user new `govuk-link--inverse` class

### CODE REVIEW

File: src/explore-education-statistics-admin/src/components/PageHeader.tsx [hunk 1]
```diff
@@ -71,27 +82,29 @@ const PageHeader = ({ wide }: Props) => {
...
+              aria-label="Top Level Navigation"
+              className="govuk-header__navigation "
             >
-              Menu
-            </button>
-            <nav>
+              <button
+                type="button"
+                className={classNames(
+                  'govuk-header__menu-button',
```
Comment (luke-h1) [inline src/explore-education-statistics-admin/src/components/PageHeader.tsx:92]: Just wondering if we can drop these classes into one string to be a bit more succinct? i.e. 
`'govuk-header__menu-button govuk-js-header-toggle'`

File: src/explore-education-statistics-admin/src/components/PageHeader.tsx [hunk 2]
```diff
@@ -71,27 +82,29 @@ const PageHeader = ({ wide }: Props) => {
...
-                'govuk-js-header-toggle',
-                { 'govuk-header__menu-button--open': menuOpen },
-              )}
-              aria-controls="navigation"
-              aria-expanded={menuOpen}
-              aria-label="Show or hide Top Level Navigation"
-              onClick={() => setMenuOpen(!menuOpen)}
+            <nav
+              aria-label="Top Level Navigation"
+              className="govuk-header__navigation "
```
Comment (luke-h1) [inline src/explore-education-statistics-admin/src/components/PageHeader.tsx:87]: nitpick: stray space in this `className`

File: src/explore-education-statistics-common/src/components/AccordionSection.tsx [hunk 3]
```diff
@@ -58,78 +59,82 @@ const AccordionSection = ({
...
+              isMounted ? (
+                <button
+                  aria-controls={contentId}
+                  aria-expanded={open}
+                  className={classes.sectionButton}
+                  id={headingId}
+                  type="button"
+                  onClick={() => {
+                    if (onToggle) {
+                      onToggle(!open, id);
```
Comment (luke-h1) [inline src/explore-education-statistics-common/src/components/AccordionSection.tsx:101]: nitpick: I believe we can simplify this to `onClick={() => { onToggle?.(!open, id) }}`

File: src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleaseHelpAndSupportSection.tsx [hunk 4]
```diff
@@ -36,118 +31,54 @@ const PublicationReleaseHelpAndSupportSection = ({
...
+              <li>
+                <Link to={externalMethodology.url}>
+                  {externalMethodology.title}
+                </Link>
+              </li>
+            )}
+          </ul>
+        </>
+      )}
+      {releaseType === 'NationalStatistics' && <NationalStatisticsSection />}
```
Comment (luke-h1) [inline src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleaseHelpAndSupportSection.tsx:67]: Part of me wonders if we could create a helper function to simplify the logic of checking what the release type is and what section we should render but think its out of scope for this PR

**Comment by luke-h1** [comment]
Can you run the UI tests and attach the report please? Just think we should triple-check that these markup changes don't break anything that'll be harder to track down later down the line

**Review by ntsim — CHANGES_REQUESTED** [review]

File: src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleaseHelpAndSupportSection.tsx (hunk 4 above)
Comment (amyb-hiveit) [reply src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleaseHelpAndSupportSection.tsx:67]: Yeah we probably could, but I'd prefer to keep this PR just to govuk-frontend related changes.

**Comment by amyb-hiveit** [comment]
> Can you run the UI tests and attach the report please? Just think we should triple-check that these markup changes don't break anything that'll be harder to track down later down the line

admin
admin_and_public
admin_and_public_2
general_public

**Review by ntsim — APPROVED** [review]

**Pull request merged**